e2e/self_node_remediation_test.go

```diff
@@ -342,6 +361,25 @@ var _ = Describe("Self Node Remediation E2E", func() {
 					})
 				})
 
+				Context("OutOfService Remediation Strategy", func() {
+					var oldPodCreationTime time.Time
+
+					BeforeEach(func() {
+						remediationStrategy = v1alpha1.OutOfServiceTaintRemediationStrategy
+						oldPodCreationTime = findSnrPod(node).CreationTimestamp.Time
```

Comment:
**node** is undefined in this context, note that this Context is nested under `Describe("Control Plane Remediation", ...` 
Where node is defined in the context of  `Describe("Workers Remediation", func() { ... `